ui/src/main/resources/PhenoTips/SimilarCases.xml

```diff
@@ -1240,7 +1336,7 @@
       <parse>0</parse>
     </property>
     <property>
-      <use>onDemand</use>
+      <use>always</use>
```

Comment:
use `always` seems wrong (things may not work with "onDemand" or "currentPage", but maybe we need to refactor it some other way instead?)